x-pack/plugins/ml/server/routes/job_service.ts

```diff
@@ -897,15 +897,15 @@ export function jobServiceRoutes({ router, routeGuard }: RouteInitialization) {
   /**
    * @apiGroup JobService
    *
-   * @api {post} /internal/ml/jobs/categorization_field_examples Get categorization field examples
-   * @apiName ValidateCategoryExamples
-   * @apiDescription Validates category examples
+   * @api {post} /internal/ml/jobs/categorization_field_validation Get categorization field examples
+   * @apiName ValidateCategoryValidation
```

Comment:
You need to edit the api name in https://github.com/elastic/kibana/blob/main/x-pack/plugins/ml/scripts/apidoc_scripts/apidoc_config/apidoc.json too

Reply:
7944f53e963d5f8edbe13250403e372d77bcb7fc